Cubical/Foundations/HLevels.agda

```diff
@@ -246,6 +246,16 @@ hLevelSuc 0 A = isContr→isProp
 hLevelSuc 1 A = isProp→isSet
 hLevelSuc (suc (suc n)) A h a b = hLevelSuc (suc n) (a ≡ b) (h a b)
 
+hLevelAdd : (n : ℕ) (m : ℕ) (A : Type ℓ) → isOfHLevel n A → isOfHLevel (n + m) A
```

Comment:
This is already proved in: https://github.com/agda/cubical/blob/master/Cubical/Foundations/HLevels.agda#L254

These lemmas should really be renamed to be of the form `isOfHLevelX`... I'm not suggesting that you do it in this PR, but if someone has time a PR fixing the naming in the HLevels file would be very much appreciated. (@m-yac maybe? :-) )